crates/turbopack-node/js/src/ipc/index.ts

```diff
@@ -133,9 +136,11 @@ function createIpc<TIncoming, TOutgoing>(
           ...structuredError(error),
         });
       } catch (err) {
+        console.error("failed to send error back to rust:", err);
```

Comment:
This should only be logged in some kind of verbose/tracing mode, as I don't think it's very actionable from a user's perspective.

Reply:
I added this because previously the error outputted was something like "unexpected EOF" so while it might not be actionable I thought it might be easier to understand and if we receive feedback easier to debug, can remove it again though